### DISCUSSION TITLE
Example on how to use pseudo-element with cva?

### DISCUSSION THREAD

**Discussion by ajmalafif** [joe-bell/cva#48] [Q&A] [answered]
I was tinkering with `cva` with `tailwind` with NextJS example repo, then I realized I needed to target pseudo-elements but find myself stuck and unsure how to proceed. Any pointers or ideas? Do I have to use `css-in-js` solution for this?

**Comment by joe-bell** [comment, marked as answer]
https://tailwindcss.com/docs/hover-focus-and-other-states#before-and-after

**Reply by ajmalafif** [reply]
Hi! I was thinking of how I do that without `tailwind` and just vanilla CSS. I had to use quite some of it to achieve this:
![CleanShot 2022-10-22 at 21 36 41](https://user-images.githubusercontent.com/827167/197342243-569b0aa4-e1ff-42bd-8847-d26a00ea7bc0.gif)
<img width="175" alt="CleanShot 2022-10-22 at 21 41 29@2x" src="https://user-images.githubusercontent.com/827167/197342284-7b398865-5c5f-434f-9de4-041b1969a308.png">

**Reply by joe-bell** [reply]
cva doesn’t generate CSS, you need to do that yourself
